parse.go

```diff
@@ -136,6 +141,13 @@ func NewParser(in io.Reader, bytesToRead int64, frameChannel chan *frame.Frame)
 	var bo binary.ByteOrder = binary.LittleEndian
 	implicit := true
 
+	// Skip trying to find and parse transfer syntax based on metadata if we have skipped reading a header
+	if optSet.skipMetadataReadOnNewParserInit {
+		p.reader.SetTransferSyntax(bo, implicit)
+
+		return &p, nil
+	}
+
```

Comment:
The reason I included this check was mostly to skip the warnings from the logic below, since they will always fire when we're not reading metadata from the dataset. Another option would be to check for `optSet.skipMetadataReadOnNewParserInit` before emitting the warnings. I'm not sure which is best. There might also be other, better ways.

Reply:
I see! If log spam is the biggest concern, how about if we switch these to `debug.Log` (so they will only be logged if the binary is build with a debug build tag) [they probably been switched over to debug.Log anyway)]?

I don't feel too strongly, so we can leave it as is, but let me know what you think!